e2e_tests/tests/cluster/test_webhooks.py

```diff
@@ -270,3 +272,135 @@ def test_specific_webhook() -> None:
     assert len(responses) == 0
     responses = server2.close_and_return_responses()
     assert len(responses) == 1
+
+
+def create_default_webhook(sess: api.Session, workspaceId: Optional[int] = None) -> int:
+    webhook_trigger = bindings.v1Trigger(
+        triggerType=bindings.v1TriggerType.EXPERIMENT_STATE_CHANGE,
+        condition={"state": "COMPLETED"},
+    )
+    webhook_url = "http://localhost"
+    res = bindings.post_PostWebhook(
+        sess,
+        body=bindings.v1Webhook(
+            url=webhook_url,
+            webhookType=bindings.v1WebhookType.DEFAULT,
+            triggers=[webhook_trigger],
+            mode=bindings.v1WebhookMode.WORKSPACE,
+            name="",
+            workspaceId=workspaceId,
+        ),
+    )
+    assert res.webhook.id is not None
+    return res.webhook.id or 0
```

Comment:
nit: Should we remove `or 0` here now that `res.webhook.id` definitely can't be `None`?

Reply:
I don't think we can since `res.webhook.id` is `Optional[int]` instead of `int`